projects/services-ogc/assets/others/greyscale.style.json

```diff
@@ -0,0 +1,2306 @@
+{
+    "version": 8,
+    "name": "Positron",
+    "metadata": {
+        "mapbox:autocomposite": false,
+        "mapbox:groups": {
+            "101da9f13b64a08fa4b6ac1168e89e5f": {
+                "collapsed": false,
+                "name": "Places"
+            },
+            "a14c9607bc7954ba1df7205bf660433f": {
+                "name": "Boundaries"
+            },
+            "b6371a3f2f5a9932464fa3867530a2e5": {
+                "collapsed": false,
+                "name": "Transportation"
+            }
+        },
+        "mapbox:type": "template",
+        "openmaptiles:mapbox:owner": "openmaptiles",
+        "openmaptiles:mapbox:source:url": "mapbox://openmaptiles.4qljc88t",
+        "openmaptiles:version": "3.x",
+        "maputnik:renderer": "mbgljs",
+        "maputnik:thunderforest_access_token": ""
+    },
+    "sources": {
+        "geotest_xyz_planet_eoc": {
+            "type": "vector",
+            "url": "https://pyxis.eoc.dlr.de/tileserver/planet0-12.json"
```

Comment:
can you replace `https://pyxis.eoc.dlr.de/tileserver/planet0-12.json` with `https://geotest.eoc.dlr.de/eoc/basemap/ogc/tiles/collections/planet_eoc/tiles/EPSG:900913/metadata?f=application%2Fjson`

Reply:
Done!